spec/models/application_form_spec.rb

```diff
@@ -887,4 +887,24 @@ def application_form_with_course_option_for_provider_with(level:)
       end
     end
   end
+
+  describe '#recruited?' do
+    context 'when a candidate has been recruited' do
+      it 'returns true' do
+        recruited_application_choice = create(:application_choice, :with_recruited)
+        other_choice = create(:application_choice, :with_rejection)
+        application_form = create(:completed_application_form, application_choices: [recruited_application_choice, other_choice])
+        expect(application_form.recruited?).to be(true)
```

Comment:
not a blocker but i have a fairly strong preference for `expect(application_form).to be_recruited`